packages/sdk/connect/src/wallet.ts

```diff
@@ -1,5 +1,16 @@
 import { EIP712TypedData } from '@celo/utils/lib/sign-typed-data-utils'
-import { RLPEncodedTx } from '../../utils/signing-utils'
+import { Address, CeloTx, EncodedTransaction, RLPEncodedTx } from './types'
+
+export interface ReadOnlyWallet {
```

Comment:
doesn't look "readonly" to me.

it has remove account, it can sign... what's the read ony about ?